helm_deploy/hmpps-interventions-service/templates/_envs.tpl

```diff
@@ -13,7 +13,7 @@ env:
   - name: SPRING_PROFILES_ACTIVE
     value: "logstash"
 
-  - name: HMPPS_AUTH_BASEURL
-    value: "{{ .Values.env.HMPPS_AUTH_BASEURL }}"
+  - name: HMPPSAUTH_BASEURL
```

Comment:
Is this because

https://github.com/ministryofjustice/hmpps-interventions-service/blob/e3ccd3129c53005f99ebaf4c9bb40be9fb6f6802/src/main/resources/application.yml#L17-L19

resolves to an environment variable name without the dash?

Reply:
yes. as per https://docs.spring.io/spring-boot/docs/2.1.18.RELEASE/reference/html/boot-features-external-config.html#boot-features-external-config-relaxed-binding-from-environment-variables

Reply:
If yes, would it make sense to match the argument as well to `${hmppsauth.baseurl}` so text searches can find them at the same time?

Reply:
good idea.

Reply:
thanks for clarifying!